deployment/deployment.json

```diff
@@ -4753,6 +4753,32 @@
           }
         }
       },
+      "qidao-base": {
+        "network": "base",
+        "status": "prod",
+        "versions": {
+          "schema": "1.3.0",
+          "subgraph": "1.0.2",
```

Comment:
shouldn't the subgraph version be 1.0.0?

Reply:
Yeah, it's weird now with only being able to deploy to the studio. You can't redeploy the same version, so I had to increment each time i tested